### ISSUE TITLE
Implement OpenSearch

### ISSUE THREAD

**Issue by GRMule** [freeCodeCamp/devdocs#74]
OpenSearch (with suggestions) would allow users to use devdocs as search provider in their web browser's native search box.

This would eliminate the need for keeping the site open on a new tab, or running the app. I already have a ton of tabs open for work, and I already have my screen real estate allocated to my code editor, the browser, Firebug, ssh terminal, and FTP.  Even on a dual-head machine, yet another application open is cumbersome.

The open search suggestions could mimic the search results shown in the left-hand panel of the site, so I have the documentation always available without adding another persistent application or tab to my workspace.

Implementation is extremely easy and lightweight, here is a brief overview: http://perishablepress.com/custom-opensearch-functionality-for-your-website/

Basically, you only would have to add an XML document to the root, and a single line of code to the page's head.
